cis_v110/section_3.sp

```diff
@@ -221,7 +224,7 @@ control "cis_v110_3_14" {
     cis_item_id = "3.14"
     cis_level   = "2"
     cis_type    = "manual"
-    service     = "core"
+    service     = "OCI/Networking"
```

Comment:
What service do we use in the OCI Insights mod? Should this be `OCI/VCN` instead?